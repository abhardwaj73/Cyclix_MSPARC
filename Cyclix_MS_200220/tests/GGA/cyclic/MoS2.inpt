CELL: 29.913816134400001 0.369599135716446 6.0212984335
FD_GRID: 100 50 30  #186 65 38
KPOINT_GRID: 1 1 10
CHEB_DEGREE: 60
BC: D C P
EXCHANGE_CORRELATION: GGA_PBE
TOL_SCF: 1e-5
SMEARING: 0.001
ELEC_TEMP_TYPE: fd
PRINT_FORCES: 1
CALC_STRESS: 1
