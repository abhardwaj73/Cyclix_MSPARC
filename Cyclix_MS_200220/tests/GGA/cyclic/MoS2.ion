ATOM_TYPE: Mo
N_TYPE_ATOM: 2
PSEUDO_POT: ./Mo_GGA.psp8
COORD_FRAC:
0.943296044063221      0.05     0.05
0.943296044063221      0.55   0.55
ATOM_TYPE: S
N_TYPE_ATOM: 4
PSEUDO_POT: ./S_GGA.psp8
COORD_FRAC:
0.844448472963803   0.333333333333333                   0
0.844448472963803   0.833333333333333                   0.5
1.042143615162638   0.333333333333333                   0
1.042143615162638   0.833333333333333                   0.5
