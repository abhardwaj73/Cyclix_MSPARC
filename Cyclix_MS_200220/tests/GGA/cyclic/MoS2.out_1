***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 19:28:49                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.913816 0.369599 6.021298 
FD_GRID: 100 50 30
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 10
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/GGA/cyclic/MoS2.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.299138 (Bohr)
Mesh spacing in y-direction        :  0.208584 (Bohr)
Mesh spacing in z direction        :  0.200710 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/GGA/cyclic/MoS2.out_1
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.71 0.21 6.62
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.61 0.21 6.42
Number of atoms of type 2          :  4
Estimated total memory usage       :  2.36 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9698122367E+01        5.162E-02        212.092
2            -2.9705553215E+01        2.244E-02        55.968
3            -2.9705070384E+01        2.016E-02        56.360
4            -2.9706662496E+01        7.863E-03        55.075
5            -2.9706839911E+01        4.282E-03        54.386
6            -2.9706839370E+01        3.992E-03        54.823
7            -2.9706853541E+01        7.915E-04        54.887
8            -2.9706853289E+01        3.994E-04        53.292
9            -2.9706853316E+01        1.988E-04        53.277
10           -2.9706854211E+01        1.053E-04        54.014
11           -2.9706855973E+01        5.135E-05        52.899
12           -2.9706854579E+01        2.841E-05        52.587
13           -2.9706854899E+01        2.255E-05        52.169
14           -2.9706855834E+01        1.353E-05        51.958
15           -2.9706855594E+01        5.416E-06        50.430
Total number of SCF: 15    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9706855594E+01 (Ha/atom)
Total free energy                  : -1.7824113356E+02 (Ha)
Band structure energy              : -4.2586582149E+01 (Ha)
Exchange correlation energy        : -2.4708138523E+01 (Ha)
Self and correction energy         : -4.2392135036E+02 (Ha)
Entropy*kb*T                       : -2.4288228288E-07 (Ha)
Fermi level                        : -1.7061659588E-01 (Ha)
Average force                      :  1.4449897917E-02 (Ha/Bohr)
Maximum force                      :  2.1655377921E-02 (Ha/Bohr)
Time for force calculation         :  10.595 (sec)
Stress (Ha/Bohr)                   :
  1.7505413591E-01 
Stress equiv. all periodic (GPa)   :
  9.7108784122E-01 
Max stress                         :  1.7505413591E-01 (Ha/Bohr)
Max stress equiv. all periodic     :  9.7108784122E-01 (GPa)
Time for stress calculation        :  11.323 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1008.583 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
