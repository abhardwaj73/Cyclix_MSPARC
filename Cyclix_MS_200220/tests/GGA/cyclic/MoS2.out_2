***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 20:34:06                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.913816 0.369599 6.021298 
FD_GRID: 100 50 30
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 10
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/GGA/cyclic/MoS2.out_2
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.299138 (Bohr)
Mesh spacing in y-direction        :  0.208584 (Bohr)
Mesh spacing in z direction        :  0.200710 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/GGA/cyclic/MoS2.out_2
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.71 0.21 6.62
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.61 0.21 6.42
Number of atoms of type 2          :  4
Estimated total memory usage       :  2.36 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
