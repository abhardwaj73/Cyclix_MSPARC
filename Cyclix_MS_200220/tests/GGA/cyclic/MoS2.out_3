***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 21:22:20                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.913816 0.369599 6.021298 
FD_GRID: 100 50 30
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 10
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/GGA/cyclic/MoS2.out_3
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.299138 (Bohr)
Mesh spacing in y-direction        :  0.208584 (Bohr)
Mesh spacing in z direction        :  0.200710 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/GGA/cyclic/MoS2.out_3
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.71 0.21 6.62
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.61 0.21 6.42
Number of atoms of type 2          :  4
Estimated total memory usage       :  2.36 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9694394216E+01        5.303E-02        209.553
2            -2.9701296177E+01        2.931E-02        55.112
3            -2.9700801335E+01        2.773E-02        55.861
4            -2.9703628723E+01        7.402E-03        55.220
5            -2.9703746829E+01        4.418E-03        54.622
6            -2.9703751887E+01        3.317E-03        54.143
7            -2.9703762456E+01        1.045E-03        53.844
8            -2.9703766949E+01        4.316E-04        53.908
9            -2.9703765629E+01        1.392E-04        53.588
10           -2.9703767217E+01        1.349E-04        53.420
11           -2.9703766630E+01        7.195E-05        52.316
12           -2.9703765082E+01        4.101E-05        52.172
13           -2.9703765756E+01        2.392E-05        52.426
14           -2.9703765222E+01        1.395E-05        51.192
15           -2.9703764791E+01        6.484E-06        49.810
Total number of SCF: 15    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9703764791E+01 (Ha/atom)
Total free energy                  : -1.7822258875E+02 (Ha)
Band structure energy              : -4.2569979837E+01 (Ha)
Exchange correlation energy        : -2.4718369875E+01 (Ha)
Self and correction energy         : -4.2392448797E+02 (Ha)
Entropy*kb*T                       : -2.4005957618E-07 (Ha)
Fermi level                        : -1.6908271764E-01 (Ha)
Average force                      :  4.5358342708E-02 (Ha/Bohr)
Maximum force                      :  6.5651487441E-02 (Ha/Bohr)
Time for force calculation         :  10.576 (sec)
Stress (Ha/Bohr)                   :
  1.2328484107E-01 
Stress equiv. all periodic (GPa)   :
  6.8390506482E-01 
Max stress                         :  1.2328484107E-01 (Ha/Bohr)
Max stress equiv. all periodic     :  6.8390506482E-01 (GPa)
Time for stress calculation        :  11.736 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1000.759 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
