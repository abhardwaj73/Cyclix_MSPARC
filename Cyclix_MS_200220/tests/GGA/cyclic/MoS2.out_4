***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 06-May-2020  Start time: 22:13:00                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.913816 0.369599 6.021298 
FD_GRID: 100 50 30
FD_ORDER: 12
BC: D C P
KPOINT_GRID: 1 1 10
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 60
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-05
TOL_POISSON: 1.00E-07
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/GGA/cyclic/MoS2.out_4
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.299138 (Bohr)
Mesh spacing in y-direction        :  0.208584 (Bohr)
Mesh spacing in z direction        :  0.200710 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  ../tests/GGA/cyclic/MoS2.out_4
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ./Mo_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.71 0.21 6.62
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ./S_GGA.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.61 0.21 6.42
Number of atoms of type 2          :  4
Estimated total memory usage       :  2.36 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9685592042E+01        4.993E-02        210.286
2            -2.9693618372E+01        3.481E-02        56.089
3            -2.9693926659E+01        2.976E-02        57.020
4            -2.9694730998E+01        6.890E-03        56.128
5            -2.9694711578E+01        6.115E-03        55.230
6            -2.9694702967E+01        4.988E-03        54.565
7            -2.9694789393E+01        2.502E-03        54.738
8            -2.9694800201E+01        3.094E-04        54.602
9            -2.9694800757E+01        2.121E-04        52.775
10           -2.9694800438E+01        1.124E-04        52.475
11           -2.9694800378E+01        6.545E-05        52.493
12           -2.9694798842E+01        5.315E-05        52.508
13           -2.9694799507E+01        3.939E-05        51.996
14           -2.9694799457E+01        2.966E-05        51.650
15           -2.9694798788E+01        2.141E-05        50.070
16           -2.9694798551E+01        1.563E-05        49.938
17           -2.9694798947E+01        8.269E-06        49.542
Total number of SCF: 17    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9694798947E+01 (Ha/atom)
Total free energy                  : -1.7816879368E+02 (Ha)
Band structure energy              : -4.2610925512E+01 (Ha)
Exchange correlation energy        : -2.4750355442E+01 (Ha)
Self and correction energy         : -4.2392727317E+02 (Ha)
Entropy*kb*T                       : -1.6426899579E-07 (Ha)
Fermi level                        : -1.9252249411E-01 (Ha)
Average force                      :  8.2176798973E-02 (Ha/Bohr)
Maximum force                      :  1.1455632162E-01 (Ha/Bohr)
Time for force calculation         :  10.536 (sec)
Stress (Ha/Bohr)                   :
  2.7001867644E-01 
Stress equiv. all periodic (GPa)   :
  1.4978900797E+00 
Max stress                         :  2.7001867644E-01 (Ha/Bohr)
Max stress equiv. all periodic     :  1.4978900797E+00 (GPa)
Time for stress calculation        :  11.661 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1106.471 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
