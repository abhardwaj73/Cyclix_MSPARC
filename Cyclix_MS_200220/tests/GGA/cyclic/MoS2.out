***************************************************************************
*                           SPARC (Feb 24, 2019)                          *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                 Start time: Wed May  6 18:48:10 2020                    *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 29.9138161344 0.369599135716446 6.0212984335 
TWIST_ANGLE: 0.000000 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 186 65 38
FD_ORDER: 12
BC: D C P 
KPOINT_GRID: 1 1 10
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 1
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001
CHEB_DEGREE: 80
NSTATES: 36
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 6
NP_BAND_PARAL: 12
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 9 4 2
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  72
Mesh spacing in x-direction        :  0.160827 (Bohr)
Mesh spacing in y-direction        :  0.160449 (Bohr)
Mesh spacing in z direction        :  0.158455 (Bohr)
Number of symmetry adapted k-points:  6
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  6
Total number of electrons          :  52
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  ././Mo_GGA.psp8
Pseudocharge radii of atom type 1  :  6.59 0.22 6.66 (x, y, z dir)
Number of atoms of type 1          :  2
Fractional coordinates of atoms of type 1     :
      0.9432960441       0.0000000000       0.0000000000
      0.9432960441       0.5000000000       0.5000000000
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  ././S_GGA.psp8
Pseudocharge radii of atom type 2  :  6.75 0.22 6.66 (x, y, z dir)
Number of atoms of type 2          :  4
Fractional coordinates of atoms of type 2     :
      0.8444484730       0.3333333333       0.0000000000
      0.8444484730       0.8333333333       0.5000000000
      1.0421436152       0.3333333333       0.0000000000
      1.0421436152       0.8333333333       0.5000000000
Estimated total memory usage       :  9.02 GB
Estimated memory per processor     :  128.23 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9696942307E+01        5.291E-02        57.463
2            -2.9705104880E+01        2.496E-02        11.018
3            -2.9703602912E+01        3.486E-02        10.820
4            -2.9706767838E+01        7.196E-03        11.223
5            -2.9706824702E+01        4.547E-03        10.704
6            -2.9706782537E+01        3.983E-03        10.616
7            -2.9706835084E+01        2.795E-03        10.844
8            -2.9706840135E+01        4.653E-04        10.622
9            -2.9706840007E+01        2.023E-04        10.604
10           -2.9706840167E+01        7.764E-05        10.598
11           -2.9706840021E+01        2.489E-05        10.510
12           -2.9706840133E+01        7.854E-06        10.486
13           -2.9706840098E+01        4.133E-06        10.526
14           -2.9706840207E+01        2.285E-06        10.415
15           -2.9706840066E+01        8.674E-07        10.397
Total number of SCF: 15    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9706840066E+01 (Ha/atom)
Total free energy                  : -1.7824104040E+02 (Ha)
Band structure energy              : -4.2583728737E+01 (Ha)
Exchange correlation energy        : -2.4707950081E+01 (Ha)
Self and correction energy         : -4.2392018443E+02 (Ha)
Entropy*kb*T                       : -1.2960591679E-07 (Ha)
Fermi level                        : -1.7122338394E-01 (Ha)
Atomic forces (Ha/Bohr):
  8.5848158106E-03  -3.0009082627E-03  -3.1396831240E-07
  2.1542853230E-02   7.4552757362E-04   3.8451546436E-08
 -1.9149763340E-02   2.4198371680E-03   1.1983751596E-07
 -2.0750059519E-02  -1.7044485656E-04   1.4203057749E-08
 -1.9920324188E-03  -1.3190128514E-03  -5.5910296645E-08
 -1.9070861292E-03  -3.1815384422E-03   1.9738648890E-07
RMS force                          :  1.2800205558E-02 (Ha/Bohr)
Maximum force                      :  2.1555749504E-02 (Ha/Bohr)
Time for force calculation         :  0.433 (sec)
Stress (Ha/Bohr): 
  1.7393311803E-01 
Stress equiv. to all periodic (GPa): 
  9.6486915446E-01 
Maximum stress                     :  1.7393311803E-01 (a.u.)
Time for stress calculation        :  0.290 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  208.888 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
